aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -358,6 +358,54 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
 
 The default value is 2^14 (16,384).
 
+### Maximum request header size
+
+`Http2.MaxRequestHeaderFieldSize` indicates the maximum allowed size in octets of request header values. This limit applies to both name and value together in their compressed and uncompressed representations. The value must be greater than zero (0).
+
+```csharp
+public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
+    WebHost.CreateDefaultBuilder(args)
+        .UseStartup<Startup>()
+        .ConfigureKestrel((context, options) =>
+        {
+            options.Limits.Http2.MaxRequestHeaderFieldSize = 12288;
+        });
+```
+
+The default value is 8192.
+
+### Initial connection window size
+
+`Http2.InitialConnectionWindowSize` indicates how many bytes of request body data the server is willing to receive and buffer at a time aggregated across all requests (streams) per connection. Note requests are also limited by Http2.InitialStreamWindowSize. The value must be greater than or equal to 65,535 and less than 2^31.
```

Comment:
* Can we go with something like "receives and buffers" over "is willing to receive and buffter" ... we're trying to stay in the present tense whenever possible.
* "at a time" :point_right: "at one time" (slightly more specific)
* "Note requests" :point_right: "Note that" ... *however* ... we've been trying to remove "note"-type language in everything except actual `[!NOTE]`s. Can it just go ... "Requests are also limited by ..."?
* Code-fence "Http2.InitialStreamWindowSize"
* I recommend adding the number after "2^31" ...

> 2^31 (2,147,483,648).